extensions/commit-issue-parser/src/models/preferences.ts

```diff
@@ -0,0 +1,9 @@
+import { ActionType } from "./actionType";
+import { ContentFormat } from "./contentFormat";
+import { TypeMode } from "./typeMode";
+
+export type Preferences = {
```

Comment:
Not needed as Raycast automatically generates this type for you.

Reply:
The type shouldn't be `any` because the `Preferences` are generated via the `raycast-env.d.ts` file 🤔 

Example from one of my own extensions:

<img width="1572" height="366" alt="CleanShot 2025-08-05 at 09 13 17@2x" src="https://github.com/user-attachments/assets/fef9e828-766b-4ecc-bf98-80cb5a6d75a4" />

Reply:
Bit more info on this can be found here: https://developers.raycast.com/api-reference/preferences#preferences

Reply:
All good, not super clear in the docs, so room for improvement there since it's text underneath the example.

Thanks for actioning the feedback so quickly!